src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -129,6 +131,26 @@ class BleDisconnectEvent : public BleEventBase
 static chip::System::Layer sSystemLayer;
 static chip::Inet::InetLayer sInetLayer;
 static chip::Controller::PythonPersistentStorageDelegate sStorageDelegate;
+static chip::app::CommandSender * sCommandSender;
+
+namespace chip {
+namespace app {
+// TODO: This function should not look like this, find a better way!
+CommandSender * GetCommandSender()
```

Comment:
Can we pass this in instead of making it global?

Reply:
Done